### PR TITLE
etcd: fix embedded etcd custom port

### PR DESCRIPTION
**Pull request by safchain**
The port used by the client if no
server list was provided was hard coded
and not using the listen port.

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
run skydive-functional-tests-backend-elasticsearch

File: config/config.go [hunk 1]
```diff
@@ -98,7 +102,7 @@ func init() {
 	cfg.SetDefault("etcd.data_dir", "/var/lib/skydive/etcd")
 	cfg.SetDefault("etcd.embedded", true)
 	cfg.SetDefault("etcd.name", host)
-	cfg.SetDefault("etcd.listen", "127.0.0.1:12379")
+	cfg.SetDefault("etcd.listen", fmt.Sprintf("127.0.0.1:%d", etcdDefaultPort))
```
Comment (masco-enovance) [inline config/config.go]: nit: string concatenation won't be sufficient?

Comment (safchain) [reply config/config.go]: that's string and integer, so will have to convert it anyway

**Pull request merged**